Content.Client/Administration/UI/Notes/NoteEdit.xaml

```diff
@@ -8,6 +8,7 @@
             <Label Name="ExpiryLabel" Text="{Loc admin-note-editor-expiry-label}" Visible="False" />
             <HistoryLineEdit Name="ExpiryLineEdit" PlaceHolder="{Loc admin-note-editor-expiry-placeholder}"
                              Visible="False" HorizontalExpand="True" />
+            <OptionButton Name="ExpiryLengthDropdown" Visible="False" />
```

Comment:
Do wish this somehow shown the resulting end date, get admins to double-check when the note expires. But probably not that big of a concern.